lib/web_ui/test/engine/semantics/semantics_test.dart

```diff
@@ -28,7 +28,9 @@ void main() {
   internalBootstrapBrowserTest(() => testMain);
 }
 
-void testMain() {
+Future<void> testMain() async {
+  await ui.webOnlyInitializePlatform();
```

Comment:
Why call the ui method here, instead of the one exposed by the engine? (see line 15, it might be hidden by the import)

Reply:
Most tests call `webOnlyInitializePlatform`, so when I started, the first couple of tests I just went with the flow. Should we do a separate clean-up for that? Right now, I'm a little fuzzy about what tests _should_ do. Maybe they should instead use the `test_embedding.dart`, which is in my clean-up list to refactor.

Reply:
I have a feeling that tests should not use "webOnlyBlah" APIs, if we can avoid them (except for a unit test that tests that that specific method correctly initializes the platform :P). The rest of the tests should use the engine implementation (or the test_embedding, as you said)

My only justification for this is that we kinda want to delete/minimize the number of `ui.webOnlyFoo` methods, but if tests rely on them, they seem "harder to delete"?

I guess we can do a cleanup for those later, I'm sure there's many more tests that use `ui.webOnlyFoo`s than the ones that show in this PR! 😅

Reply:
Added to the list: https://github.com/flutter/flutter/issues/100394 :)